### PR TITLE
simple helper class for time-based tests

### PR DESCRIPTION
**Pull request by mrmasterplan**
TODO: docs

### CODE REVIEW

**Comment by LauJohansson** [comment]
Maybe you could give a short explanation inside the function? I can see you have "TODO: docs", but as a start - function (docstring).

File: src/atc_tools/time/TimeSequence.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        """produce descending timestamps so that first call is latest."""
+        return self.next(-1 * steps)
+
+    def next(self, steps: int = 1):
+        """produce ascending timestamps so that last call is latest."""
+        self.last = self.last + steps * self.delta
+        return self.last
+
+    def __next__(self):
+        return self.next()
```
Comment (LauJohansson) [inline src/atc_tools/time/TimeSequence.py:29]: Why have a __next__ when it only calls .next() ?

File: src/atc_tools/time/__init__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from datetime import datetime, timezone, date
+
+from .TimeSequence import TimeSequence  # noqa: F401
+
+
+def dt_utc(*args, **kwargs) -> datetime:
```
Comment (LauJohansson) [inline src/atc_tools/time/__init__.py:6]: Make function explanation

File: src/atc_tools/time/TimeSequence.py (hunk 1 above)
Comment (mrmasterplan) [reply src/atc_tools/time/TimeSequence.py:29]: because this function makes it possible to use the object in an iterator. `for t in TimeSequence()`

**Pull request merged**